tests/integration/targets/sysrc/tasks/cleanup-test-entries.yml

```diff
@@ -0,0 +1,23 @@
+---
+# #2319 help prevent failing tests from existing entries
+- name: Remove test entries from /etc/rc.conf
+  lineinfile:
```

Comment:
Wouldn't it be better to just copy the original contents of `/etc/rc.conf` and restore it afterwards instead of relying on `lineinfile` and removing specific lines which has been added?

This raises two issues IMO.
- Every new test should add lines it's changing to this `Remove test entries from /etc/rc.conf` task.
- This will not catch any additional lines which has been added indirectly because of running the tests. (This should not happen of course, but why take the unnecessary risk :))

Reply:
I've updated the block to copy the /etc/rc.conf and /boot/loader.conf contents and then write it back at the end of the test block.